README.md

```diff
@@ -70,54 +56,54 @@ cb59ea7f0bc3efb8b92de87cd88b589024d18ee7 (tag: v1.1.0) feat: JR-40 argument to r
 2e6d61dee0c4ed3a0f7f887973dbc326a487675b (tag: v1.0.0) feat: github JR-1 release action
 ```
 
-For simplification in examples I omitted jira configs 
+> For simplification jira configs are omitted in the following examples 
 
-1. `jira-versioner -t v2.1.0`
+1. `klio kojiro -t v2.1.0`
     
     Found commits:
     1. `05e5705322cc2d9daf7fb376a8c5e9cbd039b257`
-    1. `9bf13576317845cd7d10980d62afe719872ceb01`
-    1. `831e4c253829dbc12683baa5b4d494aa3524f39f`
-    1. `533569497a68f04674e43e23d06fb9c1f0b3b958`
+    2. `9bf13576317845cd7d10980d62afe719872ceb01`
+    3. `831e4c253829dbc12683baa5b4d494aa3524f39f`
+    4. `533569497a68f04674e43e23d06fb9c1f0b3b958`
     
     Found tasks:
     1. `JR-4`
-    1. `JR-13`
-    1. `JR-2`
+    2. `JR-13`
+    3. `JR-2`
     
     Results: 
-    1. New version created(if not already exists) - `v2.1.0`
-    1. If task was found in commits and exists in Jira it will set fixed version for it
+    1. New Jira release created, if it doesn't exist already - `v2.1.0`.
+    2. If a task have been found in a commit messages, and it exists in Jira it is matched with the release.
```

Comment:
Instead of:
`If a task have been found in a commit messages, and it exists in Jira it is matched with the release.`
This would be better:
`If a task has been found in a commit message and it exists in Jira, it is matched with the release.`
same sentence is repeated multiple times below